todo_app/app.py

```diff
@@ -1,19 +1,35 @@
-from flask import Flask, render_template, request, redirect, url_for
-from todo_app.data import session_items
-
-from todo_app.flask_config import Config
-
-app = Flask(__name__)
-app.config.from_object(Config())
-
-
-@app.route('/')
-def index():
-    items = session_items.get_items()
-    return render_template("index.html", items = items)
-
-@app.route('/items/new', methods=['POST'])
-def add_item():
-    title = request.form['title']
-    session_items.add_item(title)
-    return redirect(url_for('index'))
+from flask import Flask, render_template, request, redirect
+import todo_app.data.trello_items as trello
+
+from todo_app.flask_config import Config
+
+app = Flask(__name__)
+app.config.from_object(Config())
+
+@app.route("/")
+def index():
+    items = trello.get_items()
+    return render_template("index.html", items=items)
+
+@app.route("/add", methods=["POST"])
+def add():
+    trello.add_item(request.form.get("title"), request.form.get("description"))
+    return redirect("/")
+
+@app.route("/complete/<id>")
+def complete(id: str):
+    trello.complete_item(id)
+    return redirect("/")
+
+@app.route("/progress/<id>")
+def progress(id: str):
+    trello.progress_item(id)
+    return redirect("/")
+
+@app.route("/remove/<id>")

```

Comment:
Not ideal for these three routes to take GET requests. They should be POST only since they modify data.

Switching to POST will mean your HTML needs to be just slightly more complicated - use a `<form>` containing a submit button instead of an anchor `<a>`